src/cli/download_tracker.rs

```diff
@@ -181,7 +181,7 @@ impl DownloadTracker {
                         let percent = (self.total_downloaded as f64 / content_len as f64) * 100.;
                         let remaining = content_len - self.total_downloaded;
                         let eta_h = Duration::from_secs(if speed == 0 {
-                            std::u64::MAX
+                            u64::MAX
```

Comment:
Should really inline the clippy fixes into the commit that originated them (maybe try https://github.com/tummychow/git-absorb).

Reply:
@djc TIL about git-absorb! I will have a try on this one. However the std::u64::MAX wasn't introduced in this patch at all, so it cannot be squashed.

Reply:
Never mind, looks like it has been done elsewhere, so I can safely drop it.

Reply:
@djc Thanks a lot for the recommendation! `git-absorb` literally saved by day :)